### PR TITLE
extensions/Dockerfile: Use fedora:38 as builder

### PR DESCRIPTION
**Pull request by LorbusChris**

### CODE REVIEW

File: extensions/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ RUN if [[ -n "${VARIANT}" ]]; then MANIFEST="manifest-${VARIANT}.yaml"; EXTENSIO
 ## Creates the repo metadata for the extensions & builds the go binary.
 ## This uses Fedora as a lowest-common-denominator because it will work on
 ## current p8/s390x.  See https://github.com/openshift/os/issues/1000
-FROM quay.io/fedora/fedora:37 as builder
+FROM quay.io/fedora/fedora:latest as builder
```
Comment (travier) [inline extensions/Dockerfile:23]:
```suggestion
FROM quay.io/fedora/fedora:38 as builder
```

**Review by travier — COMMENTED** [review]
We keep things to a fixed release so that it sticks for release branches

File: extensions/Dockerfile (hunk 1 above)
Comment (LorbusChris) [reply extensions/Dockerfile]: Makes sense. Updated

**Review by travier — APPROVED** [review]
/lgtm

**Pull request merged**